docs/references/express/overview.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+  ### Create a Clerk application
+
+  You need to create a Clerk application in the Clerk Dashboard before you can set up `@clerk/express`. For more information on setting up a Clerk application, check out the [dedicated guide](/docs/quickstarts/setup-clerk).
+
+  ### Install `@clerk/express`
+
+  To get started using Clerk with Express.js, add the [SDK](https://www.npmjs.com/package/@clerk/express) to your project:
+
+  <CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+    ```sh filename="terminal"
+    npm install @clerk/express
+    ```
+
+    ```sh filename="terminal"
+    yarn add @clerk/express
+    ```
+
+    ```sh filename="terminal"
+    pnpm add @clerk/express
+    ```
+  </CodeBlockTabs>
+
+  ### Set environment variables
+
+  <SignedIn>
+    Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+  </SignedIn>
+
+  <SignedOut>
+    1. Navigate to the Clerk Dashboard.
+    1. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+    1. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+    1. Paste your keys into your `.env.local` file.
+
+    The final result should resemble the following:
+  </SignedOut>
+
+  ```sh filename=".env.local"
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+</Steps>
+
+## Available methods
+
+### `clerkMiddleware()`
+
+The `clerkMiddleware()` helper integrates Clerk authentication into your Express application. It is required to be set in the middleware chain before using other Clerk utilities, such as `requireAuth` and `getAuth()`. It is re-exported from the `@clerk/nextjs/server` package, so see the [Next.js reference documentation](https://clerk.com/docs/references/nextjs/clerk-middleware) for more information.
+
+```js
+import { clerkMiddleware } from '@clerk/express'
+```
+
+### `requireAuth`
+
+The `requireAuth` helper protects routes in your Express application by checking if the user is authenticated or authorized. See the [reference documentation](/docs/references/express/require-auth) for more information.
```

Comment:
It feels weird to me that `requireAuth` has a dedicated page to explain it, but `getAuth()` does not, despite the content being about the same length. I think my recommendation would be to inline the `requireAuth` info on this page honestly